test-server/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN apt-get -y install crate
 ADD crate-backup.tar.gz /var/lib/crate
 
 # Expose default Crate ports
-EXPOSE 4200 4300
+EXPOSE 4200 4300 5432-5532
```

Comment:
The corresponding line in README for `docker run` should also be changed and we should note if you're already running PostgreSQL on port 5432 you'll need to either stop it or bind to a different port.